cmd/template_gitcredential.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+	generator "github.com/uselagoon/build-deploy-tool/internal/generator"
+	"github.com/uselagoon/build-deploy-tool/internal/helpers"
+	"github.com/uselagoon/build-deploy-tool/internal/lagoon"
+)
+
+var templateGitCredential = &cobra.Command{
+	Use:     "git-credential",
+	Aliases: []string{"gitcred", "gc"},
+	Short:   "Create a git credential file",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		gitURL, err := TemplateGitCredential("/home/.git-credentials")
```

Comment:
So just two things here
1 - I'd store the `/home/.git-credentials` in a const
2 - I don't know if it makes sense (I think it does) but it might make sense to have this as an argument that could be passed in - that way we can explicitly say in `build-deploy.sh` where this store file should end up, and it's not just a magic string stored between the calling code and the golang

Reply:
for (1) I'm thinking that's the default, stored in a const.

Reply:
I'll change it to a flag then, pretty simple addition.